htdocs/mri/jiv/get_file.php

```diff
@@ -92,12 +92,24 @@
     exit(4);
 }
 
+// If $File contains "DCM_", prefix automatically inserted by the
+// LORIS-MRI pipeline, identify it as $FileExt: "DICOMTAR"
+if (strpos($File, "DCM_") ) {
+    $FileExt = "DICOMTAR";
+}
+
+
 switch($FileExt) {
 case 'mnc':
     $FullPath         = $mincPath . '/' . $File;
     $MimeType         = "application/x-minc";
     $DownloadFilename = basename($File);
     break;
+case 'DICOMTAR':
+    $FullPath         = $imagePath . '/' . $File;
```

Comment:
I think it would make sense to have this:

```
$pipeline   = $config->getSetting('imaging_pipeline');
$tarchivePath = $pipeline['tarchiveLibraryDir'];
```
then your line 169 would become:
```    $FullPath         = $tarchivePath . '/' . $File;```